source/_docs/ecosystem/scenegen.markdown

```diff
@@ -60,7 +60,7 @@ optional arguments:
 
 For basic operation just supply the url and optionally the api key (using the --key option) on the command line and scenegen will output a list of all lights and switches with their attributes. Optionally use the `--scenename` flag to explicitly set the scenename.
 
-```
+```text
```

Comment:
Should this be ``bash``, as it contains a bash command?

Reply:
Adjusted (fixed some more occasions).